glide.yaml

```diff
@@ -1,6 +1,7 @@
 package: github.com/openshift/library-go
 import:
 - package: k8s.io/apimachinery
+  version: kubernetes-1.13.4
```

Comment:
how the heck was this not already pinned? do you know how long it took me to figure out this was my problem? I've gotten so dumb.

Reply:
> how the heck was this not already pinned? do you know how long it took me to figure out this was my problem? I've gotten so dumb.

Clearly that guy should be fired.